### DISCUSSION TITLE
comefrom statement

### DISCUSSION THREAD

**Discussion by CryoMyst** [dotnet/csharplang#5939] [Language Ideas] [answered]

**Comment by ChaseDRedmon** [comment]
bruh

**Comment by CassOnMars** [comment]
bruh

**Comment by Shidesu** [comment]
bruh

**Comment by Zaneris** [comment]
LGTM

**Reply by Shidesu** [reply]
Could you develop? Your point of view is interesting.

**Comment by YuKitsune** [comment]
bruh

**Comment by 333fred** [comment, marked as answer]
For future readers: this is the csharp discord engaging in some friendly trolling. My answer here is the same as on there: No.

**Comment by CryoMyst** [comment]
@333fred You were not on my mind when writing this, we already have `goto` so imagine how much neater and more readable we could make code with this new statement!

**Comment by Kuinox** [comment]
bruh

**Comment by CyrusNajmabadi** [comment]
assigned to @333fred as the obvious champion here.

**Reply by CryoMyst** [reply]
Good choice

**Comment by HaloFour** [comment]
Funny, "comefrom" was recently mentioned in this talk:

https://www.youtube.com/watch?v=vcFBwt1nu2U

**Comment by MillKaDe** [comment]
In 2011, Jon Skeet posted his EduAsync series (explaining the inner workings of async).

There is a blog post about [ComeFrom.](https://codeblog.jonskeet.uk/2011/11/11/eduasync-part-15-implementing-comefrom-with-a-horrible-hack/) ...

**Comment by Unknown6656** [comment]
bruh

**Comment by ctigrisht** [comment]
I want this

**Comment by mattwar** [comment]
Didn't even wait for April.

**Reply by CyrusNajmabadi** [reply]
He camefrom there @mattwar.
